src/assets/source/styles/base/_base.scss

```diff
@@ -23,18 +23,3 @@ body {
 
   @include theme-font(normal);
 }
-
```

Comment:
Duplicated code (`src/assets/source/styles/base/_layout.scss`); at least more or less, I have yet to adapt the details